packages/host/app/components/operator-mode/container.gts

```diff
@@ -259,7 +265,18 @@ export default class OperatorModeContainer extends Component<Signature> {
   // we might drop writes from different stack items that want to save
   // at the same time
   private write = task(async (card: Card) => {
-    return await this.cardService.saveModel(card);
+    let token = waiter.beginAsync();
+    try {
+      let savedCard = await this.cardService.saveModel(card);
+      // only do this in test env--this makes sure that we also wait for any
+      // interior card instance async as part of our ember-test-waiters
+      if (environment === 'test') {
```

Comment:
[`isTesting`](https://github.com/embroider-build/embroider/tree/main/packages/macros#istesting-isdevelopingapp) might be the newer idiomatic way to check for this?